libs/langchain/langchain/document_loaders/parsers/docai.py

```diff
@@ -24,13 +24,21 @@
 
 @dataclass
 class DocAIParsingResults:
-    """A dataclass to store DocAI parsing results."""
+    """A dataclass to store Document AI parsing results."""
 
     source_path: str
     parsed_path: str
 
 
 class DocAIParser(BaseBlobParser):
+    """`Google Cloud Document AI` parser.
+
+    For a detailed explanation of Document AI, refer to the product documentation.
+    https://cloud.google.com/document-ai/docs/overview
+    """
+
+    _client: "DocumentProcessorServiceClient"
```

Comment:
this isn't a pydantic object so don't think we want this here (unless you intended to add a class attribute instead of an instance attribute)

Reply:
That was mostly for legibility, I can remove it.